app/jobs/cache/record_counts/recent_additions_job.rb

```diff
@@ -35,14 +40,30 @@ def recent_additions(params = {})
             recent_additions << {
               label: field['label'],
               count: field['count'],
-              from: time_from,
-              query: time_range_query
+              from: time[:from],
+              query: time[:range_query]
             }
           end
         end
 
         recent_additions
       end
+
+      def recent_additions_time_now
+        @recent_additions_time_now ||= Time.now
+      end
+
+      def recent_additions_months_ago_time(months_ago)
```

Comment:
Assignment Branch Condition size for recent_additions_months_ago_time is too high. [26/15]